db/migrate/20171003201913_create_stations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateStations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :stations do |t|
+      t.text      :name
+      t.integer   :dock_count
+      t.text      :city
```

Comment:
The iteration 1 spec has this hint: "Any data that will be repeated in a significant amount of rows in your stations table should probably be extracted to its own table."

I'm wondering if it'd be worth extracting city to a separate table and using a foreign key in the stations table. 
- add column in the stations for the city_id
- create a cities table with an id and name
- model for the cities table that adds the  ```has_many :stations``` functionality

@randy-darsh @aziobrow What do y'all think?

Reply:
I am somehow lost in our own project and can't find the csv files  to reference, but it makes sense that there would be multiple stations per city.  To add to that (again, I can't find the csv file data to verify), but it's also possible that dock_count may be repetitive (e.g. maybe many stations are standardly built with 20 bike docks). At any rate, there's only so many bike docks a station can hold, so it stands to reason.  Also, since we have to do a bunch of analysis related to number of bikes per station, it might be handy to have functionality where we can see that these 10 stations all have the maximum number of docks(for example). I like it!

Reply:
I hadn't considered dock_count's potential repetitiveness, good call.

We haven't seeded the database or added the csv files to our project yet, so there's nothing to reference in the project yet (it2 adds the csv and seeds the database).

i downloaded the 660 mb file from here https://www.kaggle.com/benhamner/sf-bay-area-bike-share to look at stations locally.

Reply:
While moving `city` out to it's own table would normalize this table further, I actually believe it's somewhat premature to do that. I think it's important that you're having this discussion, and someone on a future team of yours may disagree, but in this particular case I think moving `city` out to its own table would be normalizing in anticipation of some future functionality. This would be fine, except developers are notoriously bad at planning for the future. It's not unlikely that six months from now you would have a `city` table sitting with only two columns causing your teammates to wonder why you have a whole separate table for `cities` with only that info on it.

If you had more information about a city (population, demographic info, etc.) that you wanted to store about a city (that was going to be repeated in this table) I would feel differently, but currently I don't see a strong argument to move it.